README.md

```diff
@@ -0,0 +1,112 @@
+stm32flash
```

Comment:
This was done for github only, to make a project good looking from repository. Sure, duplicating info from man can be an overhead